.github/workflows/generate-smoke-tests-list/action.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+name: generate-smoke-tests-list
+description: Generate smoke tests list
+
+inputs:
+  vault-url:
```

Comment:
I know kebab-case is actually the GH convention for naming inputs. But in the apm-pipeline-library actions, we mainly use camelCase. 
Would it make sense to align this?

Reply:
I just raised an internal GH issue so we can agree, as a team, what to do with these conventions